website/client/components/inventory/equipment.vue

```diff
@@ -65,12 +65,15 @@
           v-for="(label, group) in gearTypesToStrings",
           :key="group",
           :item="flatGear[activeItems[group]]",
+          :showPopover="flatGear[activeItems[group]] && flatGear[activeItems[group]].key.indexOf('_base_0') === -1",
```

Comment:
If it doesn't work you might be able to use this feature to access item in the slot https://vuejs.org/v2/guide/components.html#Scoped-Slots